src/Components/IssuesControl.jsx

```diff
@@ -159,10 +159,10 @@ export function Issues() {
       return
     }
 
-    const fetchComments = ((selectedIssue) => {
+    const fetchComments = async (selectedIssue) => {
       try {
         const commentsArr = []
-        const commentsData = getComments(repository, selectedIssue.number)
+        const commentsData = await getComments(repository, selectedIssue.number)
```

Comment:
Please add comment with ref to bug.

Reply:
N/m.. i'll just note this for us to read:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await

Await is required to pause execution and wait for results, which is needed here.